Engine/DataFeeds/SubscriptionSynchronizer.cs

```diff
@@ -147,7 +149,7 @@ public TimeSlice Sync(IEnumerable<Subscription> subscriptions)
                                 : packetBaseDataCollection.Data;
 
                             BaseDataCollection collection;
-                            if (universeData.TryGetValue(subscription.Universe, out collection))
+                            if (universeData.TryGetValue(subscription.Universe.First(), out collection))
```

Comment:
should we be performing this for each universe? if we want to affirm only one universe in the list, let's use `Single()`.
Also, just now realizing the naming, perhaps in a stand-alone PR, can rename `Subscription.Universe` to `Subscription.Universes` to make it clear that it's a list.